### ISSUE TITLE
`mdl-card--expand` at title level

### ISSUE THREAD

**Issue by unindented** [developit/preact-mdl#6]
I was trying to reproduce the following example from the documentation:

``` html
<div class="demo-card-image mdl-card mdl-shadow--2dp">
  <div class="mdl-card__title mdl-card--expand"></div>
  <div class="mdl-card__actions">
    <span class="demo-card-image__filename">Image.jpg</span>
  </div>
</div>
```

I was doing the following:

``` html
<Card className='demo-card-image' shadow='2'>
  <Card.Title expand></Card.Title>
  <Card.Actions>
    <span className='demo-card-image__filename'>Image.jpg</span>
  </Card.Actions>
</Card>
```

Which results in:

``` html
<div shadow="2" class="mdl-card demo-card-image mdl-shadow--2dp">
  <div expand="true" class="mdl-card__title mdl-card__title--expand"></div>
  <div class="mdl-card__actions">
    <span class="demo-card-image__filename">Image.jpg</span>
  </div>
</div>
```

The problem is that `expand` gets converted into `mdl-card__title--expand` instead of `mdl-card--expand`...

Is there a way to generate the right markup, without resorting to `className`?

**Comment by developit** [comment]
Hey @unindented - I think we'll have to special-case that class, I'm not seeing any other cases in the MDL documentation where a BEM modifier is used on another block type like that.

**Comment by unindented** [comment]
That's a bummer. It's just a `flex-grow: 1`, so I ended up adding it to my styles.

**Comment by developit** [comment]
Just released `1.1.1`, which makes your first example (the nice looking one) work out-of-the-box.
Demo:
https://jsfiddle.net/developit/x1mr6ey1/

**Comment by unindented** [comment]
You rock.
